ue4docker/infrastructure/BuildConfiguration.py

```diff
@@ -126,10 +139,9 @@ def __init__(self, parser, argv):
 			
 			# Validate the specified version string
 			try:
-				ue4Version = semver.parse(self.args.release)
-				if ue4Version['major'] != 4 or ue4Version['prerelease'] != None:
+				self.release = semver.VersionInfo.parse(self.args.release)
+				if self.release.major != 4 or self.release.prerelease is not None:
 					raise Exception()
-				self.release = semver.format_version(ue4Version['major'], ue4Version['minor'], ue4Version['patch'])
```

Comment:
This appears to have changed the format of `self.release`, but the same hasn't been done for the 'custom' side, which will do the wrong thing in the changed `_generateWindowsConfig`, I assume, as for custom builds that will be comparing a string `< vs_min_supported_unreal`.

Maybe if we need to actually track/mess with UE4 version for more than stringifying, I'd suggest we store that in a separate attribute, and either require the user to specify the base UE4 version when they do a custom-source build (not a bad idea, I think, it'd simplify other things), or somehow detect the base UE4 version, perhaps using `git fetch` to pull the Build.version file from the supplied repo.

Reply:
Yep, i broke 'custom' branch. Thanks for spotting. Fixed.